compiler/passes/insertWideReferences.cpp

```diff
@@ -2203,6 +2358,34 @@ static void fixAST() {
         }
         else if (Type* wide = wideRefMap.get(act->typeInfo())) {
           insertWideTemp(wide, act);
+        } else if (act->isRef()) {
```

Comment:
I observe that the brace-style for the new else-if statements differs from existing style.
More 'note' than 'take action'.